portfolio/src/main/webapp/script.js

```diff
@@ -26,3 +26,31 @@ function addRandomGreeting() {
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
+
+/**
+ * Cycles through images automatically
+ */
+function slideShow(ind = 0) {
+
+  if (ind > 6) {
+      ind = 0;
+  } 
+
+  // Choose image based on index
+  const imgUrl = '/images/places-' + ind + '.jpg';
```

Comment:
what's the behaviour if places-ind doesn't exist? If someone deletes places-6 in the future for instance, what would this code do? I'm wondering if there's a way to get the # of images in the folder and use that instead of hardcoding 6?